test/Chirp.Infrastructure.Tests/InMemoryDatabaseUnitTest.cs

```diff
@@ -31,7 +31,7 @@ public InMemoryDatabaseTest()
             Followers = new List<Follow>()
             };
         var testCheep = new Cheep {
-            CheepId = 1, 
+            CheepId = new Guid(1, 0, 0, new byte[]{0,0,0,0,0,0,0,0}), 
```

Comment:
I think that this is not the way you use Guid. You should use Guid.NewGuid() instead of new Guid().